doc/classes/DirAccess.xml

```diff
@@ -151,6 +154,7 @@
 			<description>
 				Returns a [PackedStringArray] containing filenames of the directory contents, excluding files. The array is sorted alphabetically.
 				Affected by [member include_hidden] and [member include_navigational].
+				[b]Note:[/b] The returned directories in the editor and after exporting in the [code]res://[/code] directory may differ as some files are converted to engine-specific formats when exported.
```

Comment:
```suggestion
				[b]Note:[/b] The returned directories in the editor and after exporting in the [code]res://[/code] directory may be different, as some files are converted to engine-specific formats when exported.
```

Reply:
Although you didn't accept this, the same note in `get_directories_at` is now ever so slightly... *different*. It's not that big of a deal, but ideally it should be consistent. You can chose between the two ways of saying it.